src/songchange/songchange_crossplatform.h

```diff
@@ -0,0 +1,136 @@
+//
+// Created by domen on 15. 10. 2018.
```

Comment:
New files need to have a license header.  The song change plugin is GPL3 according to formatter.cc, so you could just copy the header from that file:
https://github.com/audacious-media-player/audacious-plugins/blob/master/src/songchange/formatter.cc

Reply:
Should I use current year in copyright text?
/*  Audacious
 *  Copyright (C) 2005-**2018**  Audacious team
 ...